plenum/test/bls/test_bls_bft_replica.py

```diff
@@ -295,6 +315,19 @@ def test_validate_commit_incorrect_sig(bls_bft_replicas, pre_prepare_with_bls):
             assert status == BlsBftReplica.CM_BLS_SIG_WRONG
 
 
+def test_validate_commit_incorrect_signed_without_pool_state_root(bls_bft_replicas, multi_signature):
+    key = (0, 0)
+    params = create_pre_prepare_params(state_root=multi_signature.value.state_root_hash)
+    pre_prepare = PrePrepare(*params)
+    setattr(pre_prepare, f.BLS_MULTI_SIG.nm, multi_signature)
+    for sender_bls_bft in bls_bft_replicas:
+        commit = create_commit_bls_sig(sender_bls_bft, key, pre_prepare)
+        for verifier_bls_bft in bls_bft_replicas:
+            assert not verifier_bls_bft.validate_commit(commit,
+                                                        sender_bls_bft.node_id,
+                                                        pre_prepare)
+
+
```

Comment:
I think we need more unit tests when pool_state_root_hash is incorrect

Reply:
What can we add to addition the test where pool_state_root_hash in PrePrepare is incorrect and message will be discarded?